.circleci/unittest/windows/scripts/environment.yml

```diff
@@ -10,10 +10,9 @@ dependencies:
   - jpeg
   - ca-certificates
   - hdf5
-  - setuptools == 58.0.4
+  - setuptools
```

Comment:
This is a refinement as proposed in https://github.com/pytorch/vision/pull/5868#issuecomment-1107881373. See the next comment for details.